src/ucs/stats/stats.c

```diff
@@ -396,7 +396,7 @@ ucs_status_t ucs_stats_node_alloc(ucs_stats_node_t** p_node, ucs_stats_class_t *
                                  ucs_stats_node_t *parent, const char *name, ...)
 {
     ucs_stats_node_t *node;
-    ucs_stats_filter_node_t *filter_node;
+    ucs_stats_filter_node_t *filter_node = NULL;
```

Comment:
hi. thank you for PR
could you shift it into first line of variables definition? we are defining initialized variables first

Reply:
Done :+1:

Reply:
but this is initialized on line 424 in `ucs_stats_filter_node_new()` and it is not used before

Reply:
You're right. Not sure why GCC fails in compiling it though. I tested with GCC 9.2.0 on our system.